__tests__/app.test.js

```diff
@@ -79,6 +79,7 @@ describe("GET /api/articles/:article_id", () => {
           expect(article.title).toBe("Living in the shadow of a great man");
           expect(article.topic).toBe("mitch");
           expect(article.author).toBe("butter_bridge");
+          expect(article.comment_count).toBe(11);
           expect(article.body).toBe("I find this existence challenging");
```

Comment:
Rather than refactoring this test, could we please have a new test for ticket 11? You will also have to change this test so that it still passes, have a look at a more flexible matcher like toMatchObject that will help you out here! 👍

Reply:
okay thanks